partiql-tests-data/fail/syntax/query/select/select.ion

```diff
@@ -61,11 +61,3 @@
         result: SyntaxFail
     },
 }
-
-{
```

Comment:
Btw for the point about optional `FROM` in `partiql-lang-kotlin`, there was an issue to track making it optional: https://github.com/partiql/partiql-lang-kotlin/issues/210. Agree with moving this test under the `partiql` namespace and making it a success test.

Reply:
Might also be worth porting some of the tests mentioned in that lang-kotlin issue:

> 2. SELECT event.id would return << {id: 'foo'} >>
3. SELECT VALUE event.id would return << 'foo' >>
4. SELECT event.id WHERE event.type = 'SAVE' would return << {id: 'foo'} >>
5. SELECT event.id WHERE event.type != 'SAVE' would return << >>

Reply:
+1, added them in the new commit.

Reply:
Cool, thanks for including them